plenum/common/messages/node_messages.py

```diff
@@ -282,10 +281,11 @@ class MessageReq(MessageBase):
     """
     Purpose: ask node for any message
     """
+    allowed_types = {LEDGER_STATUS, CONSISTENCY_PROOF, PREPREPARE,
```

Comment:
I am not saying you should not have had the test but the test could be single test file like
```
expecteds = [
Propagate.typename:(PROPAGATE, OrderedDict([
    ("request", ClientMessageValidator),
    ("senderClient", NonEmptyStringField),
])),
ViewChangeDone.typename: (VIEW_CHANGE_DONE, OrderedDict([
    ("viewNo", NonNegativeNumberField),
    ("name", NonEmptyStringField),
    ("ledgerInfo", IterableField)
]))
]
def test_has_expected_type():
   for expected in expecteds:
       assert expected.name == ...

def test_has_expected_fields():
    for expected in expecteds:
        .....

def test_has_expected_validators():
    for expected in expecteds:
        .....
```
Something like this